module/VuFind/src/VuFind/Log/LoggerAwareTrait.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Extension of \Laminas\Log\LoggerAwareTrait with some convenience methods.
+ * Extension of \Psr\Log\LoggerAwareTrait with some convenience methods.
```

Comment:
Since we're no longer extending another trait, different language might be better here:

```suggestion
 * Implementation of PSR-3 \Psr\Log\LoggerAwareTrait with some additional convenience methods.
```